modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/TelestionConfiguration.java

```diff
@@ -6,7 +6,8 @@
  * The base class for all Telestion Verticle configurations.
  * It extends {@link JsonMessage} so all configurations are also valid json classes.
  *
- * @author Cedric Bös, Pablo Klaschka, Jan von Pichowski, Ludwig Richter
+ * @author Cedric Boes (cb0s), Pablo Klaschka (@pklaschka), Jan von Pichowski (@jvpichowski),
+ * 			Ludwig Richter (@fussel178)
```

Comment:
Seems like lots of work went into the two lines of code here 😄

Reply:
Yes indeed! Don't you remember all the hours of work, which were put into this record? A true masterpiece...